dev/update_pypi_package_index.py

```diff
@@ -0,0 +1,62 @@
+"""
+A script to update the index of known PyPI packages, which is used during dependency inference to
+print warnings when unrecognized packages are inferred as depenndencies. The index is located
+at 'mlflow/pypi_package_index.json'.
+
+# How to run (make sure you're in the repository root):
+$ python dev/update_pypi_package_index.py
+"""
+
+import argparse
+import json
+import posixpath
+import requests
+import sys
+from datetime import datetime
+from html.parser import HTMLParser
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-p",
+        "--path",
+        help="Path to the ML package versions yaml (default: mlflow/pypi_package_index.json)",
```

Comment:
Can we also update the message?

Reply:
Whoops! Thanks for catching this. Fixed!